src/modules/rtpengine/doc/rtpengine_admin.xml

```diff
@@ -2338,15 +2338,15 @@ modparam("rtpengine", "control_cmd_tos", 144)
 	<section id="rtpengine.p.hash_algo">
 		<title><varname>hash_algo</varname> (integer)</title>
 		<para>
-			Hashing algorithm to be used in node selection algorithm. Now there are 3 possibilities: legacy
-			algorithm - 0 (very basic hash over callid), SHA1 - 1 (apply sha1 over the callid and calculate hash) or
-			CRC32 - 2 (calculate crc32 sum over the callid).
+			Hashing algorithm to be used in node selection algorithm. Now there are 3 possibilities: 
+			SHA1 - 0 (apply sha1 over the callid and calculate hash) or
```

Comment:
We probably should keep the old integer values for compatibility reasons, otherwise people cfg will break after an update.